IDEAS/Fluid/Taps/TapProfile.mo

```diff
@@ -6,7 +6,7 @@ model TapProfile
   parameter Real uniCon=1/60
     "Conversion factor to convert units from DHW profile [l/min] to [kg/s]";
   parameter String loadFile=Modelica.Utilities.Files.loadResource(
-    "modelica://IDEAS/Resources/DHW/DHW_1year_2adults.txt") annotation(Dialog(loadSelector(filter="All files (*.*)", caption="Select the DHW profile file")));
+    "modelica://IDEAS/Resources/Data/Fluid/Taps/DHW_1year_2adults.txt") annotation(Dialog(loadSelector(filter="All files (*.*)", caption="Select the DHW profile file")));
```

Comment:
I would remove the default to make sure that users don't forget to include their own data file. An example of how to include their data can be found in `IDEAS.Fluid.Taps.Examples`.

Reply:
https://github.com/open-ideas/IDEAS/pull/1393/commits/364f331f2697a877166e4659434b0c9b278f1986